recipes/conda/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "4.0.6" %}
+
+package:
+  name: conda
+  version: {{ version }}
+
+source:
+  fn: conda-{{ version }}.tar.gz
+  url: https://github.com/conda/conda/archive/{{ version }}.tar.gz
+  sha256: 4ca7ec875de13519aeb3eebc9beca23adf3c41e8852c186deb503e8bc2282c40
+
+build:
```

Comment:
I think bin/activate and bin/deactivate might need to be in the `always_include` also, but this might only be needed on certain platforms.